src/ai-bundle/src/AiBundle.php

```diff
@@ -115,9 +116,18 @@ public function loadExtension(array $config, ContainerConfigurator $container, C
         foreach ($config['store'] ?? [] as $type => $store) {
             $this->processStoreConfig($type, $store, $builder);
         }
+
         $stores = array_keys($builder->findTaggedServiceIds('ai.store'));
-        if (1 === \count($stores)) {
-            $builder->setAlias(StoreInterface::class, reset($stores));
```

Comment:
unexpected to see this being removed - was it intentional?

Reply:
Yes, as the aliases are defined for each store, I'm not sure if we need this alias anymore 🤔

Reply:
please revert this, see #355 

It should still be possible to just use `StoreInterface $store` if only store is configured - which is basically quite common, i'd say

Reply:
Fixed submitted in #358.